setup.cfg

```diff
@@ -30,7 +30,14 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    django >= 2.2
+    django>=2.2.0,<4.0
+    django-solo
+    djangorestframework~=3.12.0
```

Comment:
let's not restrict the upper range, but only enforce a minimum version. otherwise we end up in the same situation as before with vng-api-common